src/main/java/org/folio/dew/service/JobCommandsReceiverService.java

```diff
@@ -131,6 +144,34 @@ public void receiveStartJobCommand(JobCommand jobCommand, Acknowledgment acknowl
 
   private void prepareJobParameters(JobCommand jobCommand) {
     var paramsBuilder = new JobParametersBuilder(jobCommand.getJobParameters());
+
+    // TODO enrich exportType.json with value MARC_EXPORT
+    if ("MARC_EXPORT".equals(jobCommand.getExportType().getValue())) {
```

Comment:
It should not be just a String in code

Reply:
All Strings will be replaced with enum values during future implementation.